Makefile

```diff
@@ -148,6 +148,9 @@ website-start:
 test-ci: install-go
 	~/.go/bin/go test ./... -v $(TESTARGS) -timeout 120m
 
+test: 
+	~/.go/bin/go test ./... -timeout 120m
```

Comment:
that's a REALLY long timeout :)

Reply:
I said the same thing - lol, seems on avg in CI our tests go about 30minutes. I can get it to 45minutes and see what happens?

Reply:
I'm not sure I believe the CI numbers... it looks like they're done long before CI reports that they are done.   Can we try something like 30 min and see how it goes.   I really think, my average is more like 5 min when I run them without any caching.